dashboard/app/models/levels/external.rb

```diff
@@ -33,7 +33,7 @@ class External < DSLDefined
 
   # Check if the level has a hand-written submit button. Once all submit buttons are removed from markdown, this can go away.
   def has_submit_button?
-    properties['markdown'].try(:include?, 'next-stage') && properties['markdown'].try(:include?, 'submitButton')
+    properties['markdown'].try(:include?, 'next-stage', 'next-lesson') && properties['markdown'].try(:include?, 'submitButton')
```

Comment:
I think this is the best direction, though I'm not sure of this syntax. You may need to do something like ` (properties['markdown'].try(:include?, 'next-stage') || properties['markdown'].try(:include?, 'next-lesson')) && properties['markdown'].try(:include?, 'submitButton')` unfortunately

Reply:
Ah I see you're removing the buttons so if that's safe that's definitely better!